### DISCUSSION TITLE
There is No Description about 'RetryConfig' on Document

### DISCUSSION THREAD

**Discussion by gloomydumber** [ReactiveX/rxjs#7023] [Report issues other than bug]
Thank you for your contribution.

I found there is no description about `RetryConfig` on documentation ([rxjs.dev/RetryConfig](https://rxjs.dev/RetryConfig))

That page is linked from `retry` function and operator's documentation on parameters section ([retry](https://rxjs.dev/api/operators/retry))

**Comment by bodote** [comment]
whats missing is:
[RetryConfig](https://rxjs.dev/RetryConfig) consists of "count" and "delay" 
e.g.:
```javascript
retry({ count: 3, delay: 1000 }) // would retry 3 times and wait 1000 msec between the retrials
````

**Comment by jakovljevic-mladen** [comment]
This is fixed in #6943 and it's now published to [the official docs](https://rxjs.dev/api/index/interface/RetryConfig). Thanks for reporting.
